Dockerfile

```diff
@@ -68,7 +68,7 @@ ENTRYPOINT ["./starctl-service-rest"]
 
 FROM openjdk:11-jre-slim as graphqlapi
 
-EXPOSE 8080
+EXPOSE 9080
```

Comment:
I guess this makes sense. Is there an issue to point to wrt port change?
Should also be mentioned in readme of sgv2-graphqlapi at least.